.github/workflows/test_stdlib_releases_with_staging.yml

```diff
@@ -0,0 +1,63 @@
+name: Test Standard Library Releases with Staging Central
+
+on:
+  workflow_dispatch:
+
+jobs:
+  test-patch-versions:
+    name: Test Pacth Standard Library Releases
```

Comment:
```suggestion
    name: Test Patch Standard Library Releases
```

Reply:
Fixed in [c5dcf5c](https://github.com/ballerina-platform/ballerina-standard-library/pull/3134/commits/c5dcf5cafb944c36435e02f97db358ffc56e865c)